src/components/optimizerTab/Sidebar.jsx

```diff
@@ -15,40 +15,39 @@ import { Utils } from 'lib/utils'
 import { SavedSessionKeys } from 'lib/constantsSession'
 import { COMPUTE_ENGINE_CPU, COMPUTE_ENGINE_GPU_EXPERIMENTAL, COMPUTE_ENGINE_GPU_STABLE } from 'lib/constants'
 import { verifyWebgpuSupport } from 'lib/gpu/webgpuDevice'
+import { useTranslation } from 'react-i18next'
+import i18next from 'i18next'
 
 const { useToken } = theme
 const { useBreakpoint } = Grid
 
 const { Text } = Typography
 
-const computeEngineToDisplay = {
-  [COMPUTE_ENGINE_GPU_EXPERIMENTAL]: 'GPU acceleration: Enabled',
-  [COMPUTE_ENGINE_GPU_STABLE]: 'GPU acceleration: Enabled',
-  [COMPUTE_ENGINE_CPU]: 'GPU acceleration: Disabled',
-}
-
 function getGpuOptions(computeEngine) {
```

Comment:
This seems broken 
![image](https://github.com/user-attachments/assets/b74824d4-a272-45cb-9af9-96ccb26546c6)